src/Tasks/Touch.cs

```diff
@@ -222,17 +221,13 @@ public override bool Execute()
                 }
             }
 
-            // Ignore touching the disk when FailIfNotIncremental.
             if (FailIfNotIncremental)
             {
-                Log.LogErrorFromResources("Touch.Touching", file);
-                return false;
-            }
-            else
-            {
-                Log.LogMessageFromResources(messageImportance, "Touch.Touching", file);
+                Log.LogWarningFromResources("Touch.Touching", file);
             }
 
+            Log.LogMessageFromResources(messageImportance, "Touch.Touching", file);
```

Comment:
Same as earlier—isn't this a duplicate message if the warning fired?

Reply:
Thanks. Elsed.